packages/web3torrent/src/index.tsx

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import Drift from 'react-driftjs';
+
 import App from './App';
 import {web3torrent, Web3TorrentContext} from './clients/web3torrent-client';
 
 ReactDOM.render(
   <Web3TorrentContext.Provider value={web3torrent}>
     <App />
+    {process.env.REACT_APP_INCLUDE_DRIFT_CHATBOX && <Drift appId="kp3gf29uk9c3" />}
```

Comment:
How about defining the appId as the env var: `DRIFT_APPID` and removing `REACT_APP_INCLUDE_DRIFT_CHATBOX ` instead of hard coding? That would make it easier to update.

Reply:
I addressed this in f8788a2. Do we think the appId should be (changed and) stored as a secret (on netlify or circle or wherever we do that) ?